packages/apps/plugins/plugin-drawing/src/hooks/useDrawingModel.tsx

```diff
@@ -12,26 +12,26 @@ import {
   TLPageId,
   TLRecord,
 } from '@tldraw/tldraw';
-import { useEffect, useState } from 'react';
+import { useEffect, useMemo } from 'react';
 import { Transaction, YEvent } from 'yjs';
 
-import { Drawing as DrawingType } from '@braneframe/types';
+import { Text } from '@dxos/client/echo';
 
-import { DrawingModel } from '../props';
+import { DrawingModel } from '../types';
 
 /**
  * Constructs model from ECHO object.
  * Derived from tldraw example: https://github.com/tldraw/tldraw/blob/main/apps/examples/src/yjs/useYjsStore.ts
  */
-export const useDrawingModel = (object: DrawingType, options = { timeout: 250 }): DrawingModel => {
-  const [store] = useState(() => createTLStore({ shapes: defaultShapes }));
+export const useDrawingModel = (data: Text, options = { timeout: 250 }): DrawingModel => {
```

Comment:
drawing model is text because...it's yjs? if we're going to use this as a generic yjs model I think we should rename it, otherwise I think we should have a separate yjs model specific for tldraw

Reply:
Yes, I think we need to surface a YJS Document type. Added todo to text-model